addons/website/static/src/snippets/s_countdown/options.js

```diff
@@ -28,6 +33,8 @@ options.registry.countdown = options.Class.extend({
                 const message = this.endMessage || qweb.render('website.s_countdown.end_message');
                 this.$target.append(message);
             }
+            this.$target.toggleClass('hide-countdown', widgetValue === 'message_no_countdown');
+            this.updateUIEndMessage();
```

Comment:
What's that? Does not seem related to the fix

Reply:
Yes, this remains from the original fix targeting `saas-14.5` by mistake. It should be removed when targeting `14.0`, `message_no_countdown` does not yet exist in this version.

Reply:
Just to be sure, what did you chose? 2 PR or add second fix in forward port?
Don't forget to add to `up to <version>` if this is not master

Reply:
J'imagine que l'avenir nous le dira :shrug:

Reply:
I'll add the second fix in the forward port

Reply:
Do you already have that second fix? (The second issue we discussed a loooooooooooot of time?)
If yes, maybe provide it to @qsm-odoo which just r+ the PR and will handle it